keypad/keypad.c

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright (c) 2020 Raspberry Pi (Trading) Ltd.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include <stdio.h>
+#include "pico/stdlib.h"
+
+
+#define KEYPAD_NUM_ROWS           4
+#define KEYPAD_NUM_COLUMNS        4
+#define NO_KEY_PRESSED         '\0'  // Just a character that unlikely be
+                                     // present in keyboards, so it can by used
```

Comment:
"can by used" -> "can be used" ?

Reply:
Good catch! Thanks for the feedback. Fixed as part of https://github.com/raspberrypi/pico-examples/pull/542/commits/139d9814a8fd61682df7feecd78f88a21579f9a0